webots_ros2_driver/webots_ros2_driver/ros2_supervisor.py

```diff
@@ -91,33 +99,61 @@ def __spawn_urdf_robot_callback(self, request, response):
             self.get_logger().info('Ros2Supervisor can not import a URDF file without a specified "urdf_path" or "robot_description" in the URDFSpawner object.')
             response.success = False
             return response
-
         self.__insertion_robot_place.importMFNodeFromString(-1, robot_string)
         self.get_logger().info('Ros2Supervisor has imported the URDF robot named "' + str(robot_name) + '".')
-        self.__urdf_robots_list.append(robot_name)
+        self.__node_list.append(robot_name)
+        response.success = True
+        return response
+    
+    
+    def __spawn_wdt_obj_callback(self, request, response):
```

Comment:
Why `wdt` in the method name and not `mdt`?

Reply:
Somewhere along the way i seem to have swapped out wbt for mdt and wdt by accident. Ill correct that now. Apologies for the confusion. So just to be clear, this feature is to import single wbt nodes such as :
Solid{
  name "wdt_imported_node"
  translation 0 0 0.5
  children [
    Shape {
      geometry Cylinder {
        height 1
        radius 1
      }
    }
  ]
}